.github/labeler.yml

```diff
@@ -78,3 +78,16 @@ schema-tests:
   - changed-files:
       - any-glob-to-any-file:
           - schema/tests/**
+
+
+bug:
+  - title: ['bug', 'error', 'failure', 'not working']
+  - body: ['unexpected behavior', 'crash', 'stopped working']
+
+enhancement:
+  - title: ['feature request', 'improve', 'add']
+  - body: ['enhancement', 'new functionality', 'extend']
+
+question:
+  - title: ['how to', 'help', 'clarification']
+  - body: ['need help', 'question about', 'understanding']
```

Comment:
I don't see anything related in their docs - https://github.com/actions/labeler?tab=readme-ov-file#usage